src/interpreter-errors.ts

```diff
@@ -89,11 +94,20 @@ export class CallingNonFunctionValue extends RuntimeSourceError {
   }
 
   public explain() {
-    return `Calling non-function value ${stringify(this.callee)}`
+    return `Calling non-function value ${stringify(this.callee)}.`
   }
 
   public elaborate() {
-    return 'TODO'
+    const calleeVal = this.callee
+    const calleeStr = stringify(calleeVal)
+    const elabStr = `Because ${calleeStr} is not a function, you cannot run ${calleeStr}().`
+    const multStr = `If you were planning to perform multiplication by ${calleeStr}, you need to use the * operator.`
+
+    if (typeof calleeVal !== 'number') {
```

Comment:
I would suggest swapping your if statement (easier to not have to parse negatives)

Reply:
Might want to check for Infinity/NaN too.